FHEM/14_SD_WS_Maverick.pm

```diff
@@ -261,6 +232,7 @@ SD_WS_Maverick_Parse($$)
   $hash->{temp_food}=$temp_food if ($temp_food ne"");
   $hash->{temp_bbq}=$temp_bbq if ($temp_bbq ne"");
   $hash->{messageType}=$messageType;
+  $hash->{state}=SD_WS_Maverick_buildState($hash);
```

Comment:
Ui,  das sieht gefährtlich aus. Du erstellt damit ein Internal state.

Was Du da speicherst ist mir noch nicht klar. Es ist weder Text noch eine Referenz auf die Speicheradresse von SD_WS_Maverick_buildState. Ist das dann eine Kopie der sub?

Ich würde es lassen und die Funktion direkt aufrufen

Reply:
Kannst Du mir erklären, was daran "gefährlich" ist?

Nach meinem Verständnis handelt es sich hier um einen ganz normalen Funktionsaufruf. Der Wert von `hash->{state}` ist das Ergebnis der Sub, in diesem Fall also der Text.

Reply:
Du hast recht. Ich war irgenwie verwirrt. Das ist eine ganz normale Wertezuweisung.

Sorry. :(